admin/index.php

```diff
@@ -182,6 +188,7 @@
 			</section>
 			<br />
 			<a href="/" class="btn"> LOGOUT </a>
+			<?php session_destroy() ?>
```

Comment:
This results in being logged out when selecting an option in the admin page.

Would be better to add this as part of a new file (e.g. `/logout/index.php` that would have the following:

```php
<?php
session_destroy();
header('Location: /');
```

Or something similar, then change the logout button to link to `/logout/`.